Setup.md

```diff
@@ -1,89 +1,69 @@
-  * [Evaluation](scripts/evaluation.py): Can be run at any time to evaluate the performance of the systems.
-     
-### Frontend and API
-
-How to deploy a flask application can be found [here](http://flask.pocoo.org/docs/0.12/deploying/)
-
-An example frontend.wsgi file, for the api just switch frontend with api:
-
-```py
-#!/opt/anaconda3/bin/python
-import sys
-import logging
-logging.basicConfig(stream=sys.stderr)
-sys.path.insert(0,"/opt/anaconda3/lib/python3.6/site-packages/")
-
-from arxivdigest.frontend.app import app as application
-```
+This document contains instructions on how to install and deploy the arXivDigest service.  Information on the sample recommender system can be found in a [separate document](sample/).
 
-Remember to configure the settings in [config.json](config.json) especially the secret_keys. More info at [Frontend configuration](arxivdigest/frontend/README.md#configurations) and  [API configuration](arxivdigest/api/README.md#configurations)
 
-Static files should be served by a webserver for best performance, to achieve this a 'data_path' must be configured in the config. Then the webserver needs to be configured to reroute calls to '/static' to the folder named static that gets generated inside this location after the first launch. If not rerouted these files will be served through Flask.
+## Initial setup
 
-#### Development
+  0. Prerequisites:
+    * [Python 3.6+](https://www.python.org/downloads/)
+    * [MySQL server](https://www.mysql.com/)
+  1. Clone this repository to a location of your choice (will be referred to as `REPO_PATH` below).
+  2. Execute all SQL scripts under [db/](db/) in sequential order, starting with [db/database.sql](db/database.sql) then V1, V2, etc.
+  3. Run `pip install .` while inside `REPO_PATH` to install the `arxivdigest` Python package and its dependencies.
+    * If installing with the purpose of development, use the command `pip install -e .` instead, to allow editing of the installed package.
+    * If running the service under an Apache Web Server, you may need to grant access to the respective user (e.g., www-data on Ubuntu) to the installed package.
+  4. Make sure to put [config.json](/config.json) in any of the below directories and update the settings specific to your system:
+    * `~/arxivdigest/config.json`
+    * `/etc/arxivdigest/config.json`
+    * `%cwd%/config.json`
 
-The API and frontend should be started by running 'app.py' in their respective folder while developing.
 
-Make sure that port 80 is free for the frontend and 5000 is free for the API or configure the frontend and api dev_ports in [config.json](/config.json)
+## Installing updates
 
-### Core
+  1. Pull changes from this repository.
+  2. Execute any new SQL scripts in [db/](db/).
+  3. Run `pip install .` while inside `REPO_PATH` to update the package and its dependencies.
+    * If needed, check permissions for the installed package.
```

Comment:
Lists are not rendered properly on github.

Reply:
Fixed